.github/workflows/release.yml

```diff
@@ -40,3 +40,9 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ steps.app-token.outputs.token }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      
+      # Push Protobufs to Buf.build registry - To be updated
+      # - uses: bufbuild/buf-push-action@v1
+      #   with:
+      #     input: proto
+      #     buf_token: ${{ secrets.BUF_TOKEN }}
```

Comment:
Makes sense this is commented out here since release needs to be as an NPM package here, but don't understand logic of the previous workflow.